docs/install.md

```diff
@@ -1370,31 +1373,37 @@ spec:
       cpu: "400m"
       memory: "150Mi"
 EOF
+```
 
-deployment.pmem-csi.intel.com/pmem-csi.intel.com created
-
+```console
+# nodeResources is dropped when accessing as v1beta1:
 $ kubectl get deployments.pmem-csi.intel.com pmem-csi.intel.com -o jsonpath='{.apiVersion}{"\n"}{.spec}{"\n"}'
 pmem-csi.intel.com/v1beta1
 {"deviceMode":"lvm"}
+
+# nodeResources still exists and can be seen when using v1alpha1:
+$ kubectl get deployments.pmem-csi.intel.com/v1alpha1 pmem-csi.intel.com -o jsonpath='{.apiVersion}{"\n"}{.spec}{"\n"}'
```

Comment:
`kubectl get deployments.v1alpha1.pmem-csi.intel.com pmem-csi.intel.com -o jsonpath='{.apiVersion}{"\n"}{.spec}{"\n"}'` is the right syntax. I wil fix.